kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/TestUtils.java

```diff
@@ -76,4 +77,22 @@ public static boolean isValidJson(String value) {
         }
         return true;
     }
+
+    public static int compareInstalledVersions(String currentVersion, String expectedVersion) {
+        Objects.requireNonNull(expectedVersion);
+        Objects.requireNonNull(currentVersion);
+
+        String[] currentParts = currentVersion.split("\\.");
+        String[] expectedParts = expectedVersion.split("\\.");
+
+        for (int i = 0; i < expectedParts.length; i++) {
+            int currentPart = i < currentParts.length ? Integer.parseInt(currentParts[i]) : 0;
+            int expectedPart = Integer.parseInt(expectedParts[i]);
+            int comparison = Integer.compare(currentPart, expectedPart);
+            if (comparison != 0) {
+                return comparison;
+            }
+        }
+        return 0;
+    }
```

Comment:
Might be worth extracting this as `VersionComparitor implements Comparable<String>`

Reply:
done. Thanks for the suggestion!